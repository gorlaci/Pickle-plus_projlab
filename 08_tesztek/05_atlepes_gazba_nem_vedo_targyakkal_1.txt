create room r1
create room r2
setroom r2 gas true
create student s
create teacher t
create tvsz tv
create airfresher a
create camambert c
create transistor tr
create rag r

addperson s r1
addperson t r2
add tv s
add a s
add c s
add tr s
add r s
neighbour r1 r2

enter r2 s