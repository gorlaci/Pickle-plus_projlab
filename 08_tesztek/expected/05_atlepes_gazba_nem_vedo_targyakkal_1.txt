
s
location: r1
itemsInHand: tv, a, c, tr, r, 
stunRemaining: 0
Game in progress

s
location: r2
itemsInHand: 
stunRemaining: 3
Game over
